statsmodels/genmod/generalized_linear_model.py

```diff
@@ -877,6 +878,23 @@ def fit(self, start_params=None, maxiter=100, method='IRLS', tol=1e-8,
             values for gradient optimization.  Only relevant if
             `method` is set to something other than 'IRLS'.
 
+        IRLS Parameters
```

Comment:
we cannot use nonstandard titles/headers in the docstrings because they are not recognized by the doc rendering

AFAICS either use `Other Parameters` or no headers for this section just continue with parameters
section 7 https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt
AFAIK we don't use `Other Parameters` anywhere so I'm not sure how it works.